centreon/www/include/core/pathway/pathway.php

```diff
@@ -144,7 +144,7 @@
 
     if (isset($_GET["host_id"])) {
         echo '<span class="pathWayBracket" > &nbsp;&#62;&nbsp; </span>';
-        echo getMyHostName(htmlentities($_GET["host_id"], ENT_QUOTES, "UTF-8"));
+        echo htmlentities(getMyHostName((int) $_GET["host_id"]));
```

Comment:
why htmlentities ? Because int cast is sufficient and htmlentities is useless.

Reply:
the int method to cast the host ID and htmlentities to encode all characters of the host name found before displaying it in the form.

Reply:
Sorry, I missread 😄 I suggest using htmlspecialchars for more security like this suggestion :  
```suggestion
        echo htmlspecialchars(getMyHostName((int) $_GET["host_id"]), ENT_QUOTES | ENT_SUBSTITUTE, 'UTF-8');
```

Reply:
it's ok for me